VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.html

```diff
@@ -1,49 +1,60 @@
-<div class="govuk-grid-row menu" style="clear: both;">
-  <div class="half-left">
-    <input
-      type="button"
-      value="Change courts"
-      (click)="goBackToVenueSelection()"
-      id="change-venue-allocation-btn"
-      class="govuk-button govuk-button--black"
-    />
-  </div>
+<div>
+  <div class="govuk-grid-row menu" style="clear: both;">
+    <div class="half-left">
+      <input
+        type="button"
+        value="Change courts"
+        (click)="goBackToVenueSelection()"
+        id="change-venue-allocation-btn"
+        class="govuk-button govuk-button--black"
+      />
+      <input type="button" value="Filters" (click)="showFilters()" id="filters-court-rooms" class="govuk-button govuk-button--black ml5" />
```

Comment:
The hover does not exist as it does in the prototype

Reply:
I disable the button Apply Filters is no changes to filters done. But if you select some options then you will see on enable button hover effect.

Reply:
I've tried this manually and on the code it doesn't show the hover event being captured to display the number of filters selected

Reply:
I don't know about this feature? it's not in AC?